app/scripts/controller-init/confirmations/transaction-controller-init.ts

```diff
@@ -214,8 +214,6 @@ function getControllers(
     preferencesController: () => request.getController('PreferencesController'),
     smartTransactionsController: () =>
       request.getController('SmartTransactionsController'),
-    transactionUpdateController: () =>
```

Comment:
This is one of the MMI controllers, from `@metamask-institutional/transaction-update`. It was never instantiated on `main` builds.